tests/helpers.py

```diff
@@ -158,6 +158,12 @@ def _get_direct_award_project_fixture(**kwargs):
         return project
 
     @staticmethod
+    def _get_direct_award_lock_project_fixture(**kwargs):
+        fixture = BaseApplicationTest._get_fixture_data('direct_award_project_fixture.json')
+        fixture['project']['lockedAt'] = "2017-09-08T14:07:14.716154Z"
+        return fixture
```

Comment:
This isn't quite what I had in mind and we probably don't want to hardcode a random timestamp in for this fixture. It should probably just use the current time, so that it plays ball with freezegun for locking times during testing. This method doesn't need to take in `**kwargs` though, because it doesn't do anything with it, so let's at least drop that.

It would be better still to have a more general purpose mutator for this fixture so you could replace any set of values with custom ones, in which case it could take in `**kwargs` and mutate the fixture appropriately.